ide/project.dependency/src/org/netbeans/modules/project/dependency/ArtifactSpec.java

```diff
@@ -62,9 +63,10 @@ public enum VersionKind {
     private final String versionSpec;
     private final String classifier;
     private final boolean optional;
+    private final FileObject localFile;
     final T data;
 
-    ArtifactSpec(VersionKind kind, String groupId, String artifactId, String versionSpec, String type, String classifier, boolean optional, T impl) {
+    ArtifactSpec(VersionKind kind, String groupId, String artifactId, String versionSpec, String type, String classifier, boolean optional, FileObject localFile, T impl) {
```

Comment:
Am I rihgt, when I expect that the localFile represnets the artefact file (jar for example)?

Reply:
Yes.